src/tools/rustc-workspace-hack/Cargo.toml

```diff
@@ -64,7 +64,7 @@ serde = { version = "1.0.82", features = ['derive'] }
 serde_json = { version = "1.0.31", features = ["raw_value"] }
 smallvec = { version = "0.6", features = ['union', 'may_dangle'] }
 url = { version = "2.0", features = ['serde'] }
-
+syn = { version = "1.0", features = ['full'] }
```

Comment:
This shouldn't be necessary unless dist builds fail with error about duplicated `syn` crate.